--- .zprezto/runcoms/zpreztorc
+++ .zpreztorc
@@ -30,6 +30,8 @@
 # Set the Prezto modules to load (browse modules).
 # The order matters.
 zstyle ':prezto:load' pmodule \
+  'syntax-highlighting' \
+  'autosuggestions' \
   'environment' \
   'terminal' \
   'editor' \
@@ -46,7 +48,7 @@
 #
 
 # Set the query found color.
-# zstyle ':prezto:module:autosuggestions:color' found ''
+zstyle ':prezto:module:autosuggestions:color' found 'fg=cyan'
 
 #
 # Completions
@@ -145,7 +147,7 @@
 # Set the prompt theme to load.
 # Setting it to 'random' loads a random theme.
 # Auto set to 'off' on dumb terminals.
-zstyle ':prezto:module:prompt' theme 'sorin'
+zstyle ':prezto:module:prompt' theme 'steeef'
 
 # Set the working directory prompt display length.
 # By default, it is set to 'short'. Set it to 'long' (without '~' expansion)
@@ -154,7 +156,7 @@
 
 # Set the prompt to display the return code along with an indicator for non-zero
 # return codes. This is not supported by all prompts.
-# zstyle ':prezto:module:prompt' show-return-val 'yes'
+zstyle ':prezto:module:prompt' show-return-val 'yes'
 
 #
 # Python
@@ -196,8 +198,8 @@
 
 # Set syntax highlighters.
 # By default, only the main highlighter is enabled.
-# zstyle ':prezto:module:syntax-highlighting' highlighters \
-#   'main' \
+zstyle ':prezto:module:syntax-highlighting' highlighters \
+  'main' \
 #   'brackets' \
 #   'pattern' \
 #   'line' \
