--- .emacs.d/static/packages.example.el
+++ .doom.d/packages.el
@@ -47,3 +47,41 @@
 ;; (unpin! pinned-package another-pinned-package)
 ;; ...Or *all* packages (NOT RECOMMENDED; will likely break things)
 ;; (unpin! t)
+
+
+(package! centaur-tabs
+  ;; workaround https://github.com/doomemacs/doomemacs/issues/7904
+  :recipe (:build (:not compile))
+  :pin "d6009c295a4363930247ae9a4d1125aea4d3fd74")
+(package! centaur-tabs
+  :disable (not (doom-module-active-p :ui 'tabs)))
+(package! copilot
+  :recipe (:host github :repo "copilot-emacs/copilot.el"))
+(package! eaw
+  :recipe (:host github :repo "hamano/locale-eaw" :files ("dist/*.el")))
+(package! edit-indirect)
+(package! emacs-pager
+  :recipe (:host github :repo "mbriggs/emacs-pager"))
+(package! ghq)
+(package! helm-ghq)
+(package! helm-ls-git)
+(package! init-loader)
+(package! language-detection) ;; used in edit-indirect
+(package! multi-term)
+(package! multi-vterm)
+(package! nhexl-mode)
+(package! seq) ;; > Emergency (magit): Magit requires 'seq' >= 2.24,
+(package! ssh-agency
+  :ignore (not (eq system-type 'gnu/linux)))
+(package! tty-format) ;; from user42
+(package! undo-tree)
+(package! wgrep
+  :recipe (:files ("*.el")))
+(package! xclip
+  :disable (not (doom-module-active-p :os 'tty)))
+(package! yaml-path
+  :recipe (:host github :repo "gidoichi/yaml-path" :files ("emacs/yaml-path.el")))
+
+(load
+ (expand-file-name (concat "custom-" (file-name-nondirectory load-file-name)) doom-private-dir)
+ 'noerror)
